configure

```diff
@@ -915,6 +926,26 @@ def configure_node(o):
   else:
     o['variables']['node_enable_v8_vtunejit'] = 'false'
 
+  if flavor != 'linux' and (options.enable_pgo_generate or options.enable_pgo_use):
+    raise Exception(
+      'The pgo option is supported only on linux.')
+
+  if flavor == 'linux':
+    if options.enable_pgo_generate or options.enable_pgo_use:
+      if CC != 'gcc' or CXX != 'g++':
```

Comment:
This isn't a very good check, especially since some distros support multiple gcc/g++ versions using different executable names.

Reply:
Hello

Thank you for your feedback and suggestion. I will look for a better solution asap.

@octaviansoldea